Automattic/vip-block-data-api
# Add PHPDocs and add it to the phpcs workflow

## Description

This will add PHPDocs everywhere, and will enable PHPDocs validation as part of the PHPCS checks. This will partially address #41.

Another PR will be incoming in which type hinting will be added. 

## Steps to Test

There should be no tests to run or changes to check because it's only docs that have been updated.
